SPEC.md

```diff
@@ -373,7 +374,7 @@ Examples:
 * example.com/ourapp
 * sub-domain.example.com/org/product/release
 
-An AC Name Type cannot be an empty string.
+An AC Name Type cannot be an empty string and must begin and end with an alphanumeric character.
```

Comment:
Nit, could you move this to line 368. Then we have the "human" description (line 367 + this), followed by the "machine" description (regex)